gateway/auth_manager.go

```diff
@@ -74,12 +77,32 @@ func (b *DefaultSessionManager) ResetQuota(keyName string, session *user.Session
 
 	rateLimiterSentinelKey := RateLimitKeyPrefix + keyName + ".BLOCKED"
 
-	// Clear the rate limiter
-	b.store.DeleteRawKey(rateLimiterSentinelKey)
+	// Clear the rate limiter and
 	// Fix the raw key
-	b.store.DeleteRawKey(rawKey)
+	keys := []string{rateLimiterSentinelKey, rawKey}
+	for _, acl := range session.AccessRights {
+		if acl.AllowanceScope == "" {
+			continue
+		}
+		keys = append(keys, QuotaKeyPrefix+acl.AllowanceScope+"-"+keyName)
+	}
+	type clientProvider interface {
```

Comment:
we could use `b.store.DeleteKeys(keys)` to bulk delete keys

Reply:
i tried this earlier, the deletekeys adds a prefix and does not delete the raw keys, and changing the method signature would apparently break plugins